src/bors/event.rs

```diff
@@ -6,6 +6,8 @@ use octocrab::models::RunId;
 pub enum BorsRepositoryEvent {
     /// A comment was posted on a pull request.
     Comment(PullRequestComment),
+    /// When a new commit is pushed to the pull request branch.
+    PullRequestPushed(PullRequestPushed),
```

Comment:
```suggestion
    PullRequestCommitPushed(PullRequestCommitPushed),
```
or something like that? Otherwise it sounds like a PR was pushed, which does not make sense (it sounds more like a PR was created instead).